hack/test-cmd.sh

```diff
@@ -586,7 +586,7 @@ oc delete bc/ruby-sample-build-validtag
 oc delete bc/ruby-sample-build-invalidtag
 
 # Test admin manage-node operations
-[ "$(openshift admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
+[ "$(atomic-enterprise admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
```

Comment:
yes that are same, but for consistency, I recommended oadm as thats what is being used everywhere.